### DISCUSSION TITLE
Complex bignumbers?

### DISCUSSION THREAD

**Discussion by LegionMammal978** [josdejong/mathjs#2718] [Ideas]
It would be useful to have a form of complex bignumber (i.e., a pair of real bignumbers with complex operations); most math libraries I've seen support one or the other, but not both.

**Comment by josdejong** [comment]
yup, definitely. Same holds for Units for example. We've been discussing this before, I just can't find the issue right now. Thanks for your suggestion.

Related: #550

**Comment by josdejong** [comment]
I think this is a duplicate of #2709
